.github/CODEOWNERS

```
@@ -0,0 +1,4 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+
+*       @department-of-veterans-affairs/backend-review-group * @department-of-veterans-affairs/frontend-review-group
```

I don't think this will work. CODEOWNERS adds the last matching reviewer for each file on a PR. Probably will need a combined group or we can just either or the FE / BE groups. I'm fine with either. 
```suggestion
*  @department-of-veterans-affairs/frontend-review-group 
```